src/hooks/common/useSMS/index.ts

```
@@ -0,0 +1,70 @@
+    (async () => {
+      const isAvailable = await SMS.isAvailableAsync();
+      setIsLoading(true);
+
+      if (isAvailable) {
+        setIsAvailable(true);
+        const { result } = await SMS.sendSMSAsync(addresses, message, options);
+
+        if (result === "sent") {
+          console.log("SMS sent successfully");
+          setSendStatus("sent");
+          onSuccess && onSuccess();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)